brigade-worker/test/job.ts

```diff
@@ -134,5 +134,15 @@ describe("job", function() {
         assert.isFalse(j.storage.enabled);
       });
     });
+    describe("#annotations", function() {
+      beforeEach(function() {
+        j = new mock.MockJob("my-job");
+      });
+      it("is a empty list that can be written", function() {
```

Comment:
Very minor non-blocking nit: "is an empty list"

Reply:
lol, i can fix that up.